<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像调节工具栏</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>591</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_redMinText">
             <property name="text">
              <string>色相最小值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_redMinValue">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar_redMin">
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="value">
            <number>85</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_greenMinText">
             <property name="text">
              <string>饱和度最小值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_greenMinValue">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar_greenMin">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_blueMinText">
             <property name="text">
              <string>亮度最小值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_blueMinValue">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar_blueMin">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>220</number>
           </property>
           <property name="sliderPosition">
            <number>220</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_redMaxText">
             <property name="text">
              <string>色相最大值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_redMaxValue">
             <property name="text">
              <string>180</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar_redMax">
           <property name="maximum">
            <number>180</number>
           </property>
           <property name="value">
            <number>115</number>
           </property>
           <property name="sliderPosition">
            <number>115</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_greenMaxText">
             <property name="text">
              <string>饱和度最大值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_greenMaxValue">
             <property name="text">
              <string>255</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar_greenMax">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_blueMaxText">
             <property name="text">
              <string>亮度最大值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_blueMaxValue">
             <property name="text">
              <string>255</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar_blueMax">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>倾斜角度限制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="angleRange">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>最小面积限制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="minArea">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>最大高宽比</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="maxHeightWidthRat">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>最小高宽比</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="minHeightWidthRat">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>灯柱偏向角</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tanAngle">
             <property name="minimum">
              <double>0.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1.700000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
             <property name="value">
              <double>0.600000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>装甲板偏向角</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deviationAngle">
             <property name="maximum">
              <double>30.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>2.000000000000000</double>
             </property>
             <property name="value">
              <double>15.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>甲板像素均值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="armourPixelAvg">
             <property name="maximum">
              <double>150.000000000000000</double>
             </property>
             <property name="value">
              <double>60.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
